library/experiments/hate_speech.py

```diff
@@ -0,0 +1,206 @@
+from copy import deepcopy
+from typing import List
+
+from blocks.adaptors import ListOfListsToNumpy
+from blocks.augmenters.spelling_autocorrect import SpellAutocorrectAugmenter
+from blocks.concat import DataSource
+from blocks.ensemble import Ensemble
```

Comment:
I moved this from library/examples to here. this file is not new, but all the functions have been moved out